docs/_sass/custom/custom.scss

```diff
@@ -799,3 +867,21 @@ th, td {
 .tabs .ui-tabs-nav .ui-state-active a {
 	color: $teal-green;
 }
+/*overridden*/
```

Comment:
The next part doesn't look like scss - example I found: https://gist.github.com/akullpp/6375168d0400ee2e7e3a93820d94ba54